framework/contracts/account/src/contract.rs

```diff
@@ -418,15 +420,8 @@ pub fn query(deps: Deps, env: Env, msg: QueryMsg) -> StdResult<Binary> {
 
 #[cfg(feature = "xion")]
 #[cfg_attr(feature = "export", cosmwasm_std::entry_point)]
-pub fn sudo(
-    deps: DepsMut,
-    env: Env,
-    msg: abstract_xion::contract::AccountSudoMsg,
-) -> abstract_xion::error::ContractResult<Response> {
-    if let abstract_xion::contract::AccountSudoMsg::BeforeTx { .. } = &msg {
-        AUTH_ADMIN.save(deps.storage, &true)?;
-    };
-    abstract_xion::contract::sudo(deps, env, msg)
+pub fn sudo(deps: DepsMut, env: Env, msg: crate::msg::AccountSudoMsg) -> AccountResult {
+    abstract_xion::contract::sudo(deps, env, msg).map_err(Into::into)
```

Comment:
Sounds like you're breaking stuff here

Reply:
Wow, that's very breaking indeed! git didn't even show me those lines during merge conflict